pkg/images/enricher/enricher_impl.go

```diff
@@ -259,6 +266,9 @@ func (e *enricherImpl) EnrichImage(ctx context.Context, enrichContext Enrichment
 	// here.
 	if err != nil {
 		errorList.AddError(err)
+		if shouldDelegate {
+			errorList.AddError(errCentralScan)
+		}
 		return EnrichmentResult{ImageUpdated: didUpdateMetadata, ScanResult: ScanNotDone}, errorList.ToError()
 	}
```

Comment:
Thinking we may be able to reduce the 'lines' here (and tests for `shouldDelegate` here and below) by defining a variable at the top the function that starts as nil, something like:

```go
var delegateErr error
```
The error would be set when the fallback happens:

```go
...
if errors.Is(err, errox.InvalidArgs) {
   // Log the warning and try to keep enriching
   log.Warnf("No default cluster found for delegation: %v, proceed enriching with Central", err)
   delegateErr = errors.New("No cluster specified for delegated scanning and Central scan attempt failed.")
} else {
...
```

And then here can do something like:

```suggestion
	if err != nil {
		errorList.AddErrors(err, delegateErr)
		return EnrichmentResult{ImageUpdated: didUpdateMetadata, ScanResult: ScanNotDone}, errorList.ToError()
	}
```